article.md

```diff
@@ -0,0 +1,326 @@
+
+
+```
+Poetry install
+```
+
+
+
+## Building the LinkedIn Scraper using Crawlee for Python
+
+In this section, we are going to be building the scraper using the Crawlee Python package. To learn more about Crawlee, check out their [documentation](https://crawlee.dev/python/docs/quick-start).
+
+				
+
+
+### Inspecting the LinkedIn job Search Page
+
+Open LinkedIn on your web browser and sign out from the website (if you already have an account logged in). You should see an interface like this.
+
+
+
+
+![alt_text](images/image2.png "image_tooltip")
+
+
+Navigate to the jobs section, search for a job and location of your choice; copy the Uniform Resource Locator (URL).
+
+
+
+![alt_text](images/image3.png "image_tooltip")
+
+
+You should have something like this:
+
+`https://www.linkedin.com/jobs/search?keywords=Backend%20Developer&location=Canada&geoId=101174742&trk=public_jobs_jobs-search-bar_search-submit&position=1&pageNum=0` 
+
+The URL can be divided into two:
+
+
+
+* The base URL
+* The parameters
+
+The base URL is usually before the question mark while the parameters come after the question mark. Furthermore, you can change the inputs to the parameters to get different results on your webpage.
+
+Our focus is going to be on the keyword and location parameters. The job title the user supplies will serve as input to the keyword parameter while the location the user supplies will go into the location parameter. Lastly, the geoId parameter will be removed while we keep the other parameters constant.
+
+We are going to be making changes to our `main.py` file. Copy and paste the code below in your `main.py` file.
+
+
+```
+from crawlee.playwright_crawler import PlaywrightCrawler
+from .routes import router                                     
+import urllib.parse
+
+async def main(title: str, location: str, data_name: str) -> None:
+    base_url = "https://www.linkedin.com/jobs/search"
+
+    # URL encode the parameters
+    params = {
+        "keywords": title,
+        "location": location,
+        "trk": "public_jobs_jobs-search-bar_search-submit",
+        "position": "1",
+        "pageNum": "0"
+    }
+    encoded_params = urllib.parse.urlencode(params)
+    encoded_url = f"{base_url}?{encoded_params}"
+
+    # Initialize the crawler
+    crawler = PlaywrightCrawler(
+        request_handler=router,
+    )
+
+    # Run the crawler with the initial list of URLs
+    await crawler.run([encoded_url])
+
+    # Save the data in a CSV file
+    output_file = f"{data_name}.csv"
+    await crawler.export_data(output_file)
+```
+
+
+ 
+
+The main function takes in the title, location and data_name parameters. Furthermore, we used the [Urllib](https://docs.python.org/3/library/urllib.html) library to encode the base URL and its parameters into a format that can be used to make requests to the web. 
+
+Now that we have encoded the URL, the next step for us is to set up the playwright crawler to take in your custom-defined routers. 
```

Comment:
That's already done in the generated project - in fact, the next step is adjusting the generated router to handle linkedin job postings.